From 551e5c1b369b7bc9cb71521e7aaff8fcc557c7b3 Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@nxp.com>
Date: Fri, 4 Aug 2017 14:32:46 +0800
Subject: [PATCH 2328/5242] MLK-16136-7 irqchip: imx-irqsteer: correct
 registers macro defitions

commit  5af4e2d733d8a22e9da6eca6986c19a1fe82e89a from
https://source.codeaurora.org/external/imx/linux-imx.git

Correct several macro definitions related with irqsteer
to avoid incorrect expression calculation due to operators
priority.

Signed-off-by: Fancy Fang <chen.fang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/irqchip/irq-imx-irqsteer.c |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/irqchip/irq-imx-irqsteer.c b/drivers/irqchip/irq-imx-irqsteer.c
index bc16453..9727445 100644
--- a/drivers/irqchip/irq-imx-irqsteer.c
+++ b/drivers/irqchip/irq-imx-irqsteer.c
@@ -18,9 +18,9 @@
 
 #define CHANREG_OFF	(irqsteer_data->channum * 4)
 #define CHANCTRL	0x0
-#define CHANMASK(n)	(0x4 + 0x4 * n)
-#define CHANSET(n)	(0x4 + (0x4 * n) + CHANREG_OFF)
-#define CHANSTATUS(n)	(0x4 + (0x4 * n) + (CHANREG_OFF * 2))
+#define CHANMASK(n)	(0x4 + 0x4 * (n))
+#define CHANSET(n)	(0x4 + (0x4 * (n)) + CHANREG_OFF)
+#define CHANSTATUS(n)	(0x4 + (0x4 * (n)) + (CHANREG_OFF * 2))
 #define CHAN_MINTDIS	(0x4 + (CHANREG_OFF * 3))
 #define CHAN_MASTRSTAT	(CHAN_MINTDIS + 0x4)
 
-- 
1.7.9.5

